Amplitude/AMPUtils.m

```diff
@@ -141,4 +141,13 @@ + (NSDictionary *) validateGroups:(NSDictionary *) obj
     return [NSDictionary dictionaryWithDictionary:dict];
 }
 
++ (NSString*) platformDataDirectory
+{
+#if TARGET_OS_TV
+    return [NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES) objectAtIndex: 0];
```

Comment:
The cache directory is valid in tvOS, but the data stored there is only guaranteed to be there while the application is running.